test/unit/spec/common/validate2-Spec.js

```diff
@@ -3,8 +3,9 @@ define([
     'common/validation',
     'widgets/appWidgets2/validators/int',
     'widgets/appWidgets2/validators/resolver',
+    'widgets/appWidgets2/validators/constants',
```

Comment:
This seems to overlap other validators, will get removed / merged in a later PR.